### PR TITLE
Skip test_can_validate_ip_san when unsupported

### LINKED ISSUE #1117: ERROR: Ensure that urllib3 can validate SANs with IP addresses in them.
I get this error when trying to build urllib3-1.20

(This is in connection with https://github.com/NixOS/nixpkgs/issues/22616)

```
======================================================================
ERROR: Ensure that urllib3 can validate SANs with IP addresses in them.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/nix-build-python2.7-urllib3-1.20.drv-0/urllib3-1.20/test/with_dummyserver/test_https.py", line 508, in test_can_validate_ip_san
    r = https_pool.request('GET', '/')
  File "/tmp/nix-build-python2.7-urllib3-1.20.drv-0/urllib3-1.20/urllib3/request.py", line 66, in request
    **urlopen_kw)
  File "/tmp/nix-build-python2.7-urllib3-1.20.drv-0/urllib3-1.20/urllib3/request.py", line 87, in request_encode_url
    return self.urlopen(method, url, **extra_kw)
  File "/tmp/nix-build-python2.7-urllib3-1.20.drv-0/urllib3-1.20/urllib3/connectionpool.py", line 630, in urlopen
    raise SSLError(e)
SSLError: hostname '127.0.0.1' doesn't match either of 'localhost', '127.0.0.1'
-------------------- >> begin captured logging << --------------------
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): 127.0.0.1
urllib3.connection: ERROR: Certificate did not match expected hostname: 127.0.0.1. Certificate: {'subjectAltName': (('email', 'root@localhost'), ('DNS', 'localhost'), ('IP Address', '127.0.0.1')), 'notBefore': u'Dec 22 07:58:40 2011 GMT', 'serialNumber': u'01', 'notAfter': 'Dec 18 07:58:40 2021 GMT', 'version': 3L, 'subject': ((('countryName', u'FI'),), (('stateOrProvinceName', u'dummy'),), (('localityName', u'dummy'),), (('organizationName', u'dummy'),), (('organizationalUnitName', u'dummy'),), (('commonName', u'localhost'),)), 'issuer': ((('countryName', u'FI'),), (('stateOrProvinceName', u'dummy'),), (('localityName', u'dummy'),), (('organizationName', u'dummy'),), (('organizationalUnitName', u'dummy'),), (('commonName', u'SnakeOil'),), (('emailAddress', u'dummy@test.local'),))}
--------------------- >> end captured logging << ---------------------
```

Is this user error? I can't really understand the error:
`SSLError: hostname '127.0.0.1' doesn't match either of 'localhost', '127.0.0.1'` (!)

### PR DESCRIPTION
**Pull request by Lukasa**
Resolves #1117.

### CODE REVIEW

**Comment by haikuginger** [comment]
Add a pragma to catch the linting error, and it looks good to go!

File: test/with_dummyserver/test_https.py [hunk 1]
```diff
@@ -529,6 +529,11 @@ class TestHTTPS_IPSAN(HTTPSDummyServerTestCase):
 
     def test_can_validate_ip_san(self):
         """Ensure that urllib3 can validate SANs with IP addresses in them."""
+        try:
+            import ipaddress  # noqa
```
Comment (sigmavirus24) [inline test/with_dummyserver/test_https.py]: If you're using Flake8 3.x, you can use `# noqa: F...` to specify just that code.

**Comment by Lukasa** [comment]
@sigmavirus24 Addressed that.

**Comment by sigmavirus24** [comment]
🍰

**Pull request merged**